src/components/calculator/PersonRiskControls.tsx

```diff
@@ -70,6 +70,53 @@ const personRiskPopover = (
   </Popover>
 )
 
+interface DurationInputProps {
+  value: number
+  onChange: (duration: number) => void
+}
+
+const DurationInput: React.FunctionComponent<DurationInputProps> = (
+  props: DurationInputProps,
+): JSX.Element => {
+  const [hours, setHours] = useState(Math.floor(props.value / 60))
+  const [minutes, setMinutes] = useState(props.value % 60)
+  const calculateDuration = (hours: number, minutes: number) => {
+    const h = Number.isNaN(hours) ? 0 : hours
+    const m = Number.isNaN(minutes) ? 0 : minutes
+    return h * 60 + m
+  }
+  return (
+    <InputGroup className="mb-3">
+      <input
+        className="form-control form-control-lg col-md-3"
+        type="number"
+        inputMode="numeric"
+        pattern="[0-9]*"
+        value={hours}
+        onChange={(e) => {
+          const newHours = Math.max(0, parseInt(e.target.value))
+          setHours(newHours)
+          props.onChange(calculateDuration(newHours, minutes))
+        }}
+      />
+      <InputGroup.Text>Hours</InputGroup.Text>
+      <input
```

Comment:
I wonder how this would look with space between the hour entry and the minutes entry? https://react-bootstrap.github.io/components/forms/#forms-layout-grid